cli/Cargo.toml

```diff
@@ -75,3 +75,5 @@ version = "1.0"
 version = "1.21"
 features = ["rt"]
 
+[dependencies.num-bigint]
+version = "0.4.3"
```

Comment:
please keep deps sorted

Reply:
Thank you! Do you have any linter recommendations? E.g is there any vscode/vim plugin?

Reply:
So, personally for tomls I use taplo-ls with neovim + nvim-lsp. However, it doesn't sort dependencies, moreover this repo doesn't use it for autoformat so I usually disable it for saving files here :) as per this comment, just move the dep some lines above and it'll be okay